### ISSUE TITLE
[Suggestion] Option to disable autosave on quit

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#1850]
A lot of the time, when encountering a bug and wanting to duplicate the exact circumstances it occurred under, the best way to do this is to save, test those conditions, then quit and reload to see if it occurs again. Autosave on quit makes this impossible without using the task manager to force-quit the game. An option to toggle autosave & autosave on quite would be useful.

**Comment by tsunamayo** [comment]
Yes correct. I could also do a save without quit button, but thats less clean. On my to-do asap, as it is impacting tester workflow (and was already suggested I think)!
